components/brave_rewards/browser/rewards_notification_service_impl.cc

```diff
@@ -377,7 +377,8 @@ void RewardsNotificationServiceImpl::OnReconcileComplete(
   if ((converted_result == ledger::Result::LEDGER_OK &&
        converted_type == ledger::RewardsType::AUTO_CONTRIBUTE) ||
        converted_result == ledger::Result::LEDGER_ERROR ||
-       converted_result == ledger::Result::NOT_ENOUGH_FUNDS ||
+      (converted_result == ledger::Result::NOT_ENOUGH_FUNDS &&
+       converted_type != ledger::RewardsType::ONE_TIME_TIP)||
```

Comment:
nit: Might be an opportunity to break this up into something more readable?
```
bool canAutoContribute = (converted_result == ledger::Result::LEDGER_OK) &
       (converted_type == ledger::RewardsType::AUTO_CONTRIBUTE);
bool insufficientFundsForContribution = (converted_result == ledger::Result::NOT_ENOUGH_FUNDS) &
       (converted_type != ledger::RewardsType::ONE_TIME_TIP);

if (canAutoContribute || 
converted_result == ledger::Result::LEDGER_ERROR || 
insufficientFundsForContribution ||
converted_result == ledger::Result::TIP_ERROR) {
```
Or even bring each condition out into its own bool... just a thought.

Reply:
fixed